pkg/scheduler/framework/interface.go

```diff
@@ -649,6 +649,9 @@ type Framework interface {
 
 	// SetPodNominator sets the PodNominator
 	SetPodNominator(nominator PodNominator)
+
+	// Close calls Close method of each plugin.
+	Close() error
```

Comment:
Should we consider including a context here? Useful for wasm plugin? cc @sanposhiho

Reply:
Well, we [might use it](https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/blob/73a62c8e1abe3e0efe6c523610b3cc270cdcd1f0/scheduler/plugin/plugin.go#L446) to close wasm runtimes though, this `ctx` doesn't work for timeout etc as mentioned at below doc and thus I'm not sure it should be something to be passed from the K8s scheduler side. 

> 	// Note: The context parameter is used for value lookup, such as for
	// logging. A canceled or otherwise done context will not prevent Close
	// from succeeding.
https://github.com/tetratelabs/wazero/blob/498d4b4f72b05d57b5003fc59fd4045f488a2279/api/wasm.go#L224-L226

There might be other usecases like contexual logging inside `Close()` though, at least for a wasm plugin, I can say we don't need `ctx` in the parameter at the moment. (= we can consider later if it's needed.)

Reply:
SG since it's `io.Closer()` not a new Closer() interface.